src/components/Form/Form.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+import { Todo, User } from '../../types';
+
+import { InputTitle } from './InputTitle';
+import { SelectUser } from './SelectUser';
+import { getNewId } from '../../utils/getNewId';
+
+type Props = {
+  users: User[];
+  todos: Todo[];
+  onAddTodo: (todo: Todo) => void;
+};
+
+export const Form: React.FC<Props> = ({ users, todos, onAddTodo }) => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+
+  const [hasTitleError, setHesTitleError] = useState(false);
+  const [hasUserIdError, setHesUserIdError] = useState(false);
```

Comment:
There is a typo in the state variable names `setHesTitleError` and `setHesUserIdError`. They should be `setHasTitleError` and `setHasUserIdError` to match the naming convention and avoid confusion.